targets/TARGET_Cypress/TARGET_PSOC6/sb-tools/README.md

```diff
@@ -0,0 +1,150 @@
+#### Version of Python required is 3.7+
```

Comment:
@ARMmbed/mbed-os-psa Please review this readme and what sb tools do for PSA targets